crates/storage/Cargo.toml

```diff
@@ -25,8 +25,7 @@ scale = { package = "parity-scale-codec", version = "2.0", default-features = fa
 derive_more = { version = "0.99", default-features = false, features = ["from", "display"] }
 scale-info = { version = "0.5", default-features = false, features = ["derive"], optional = true }
 cfg-if = "1.0"
-array-init = "1.0"
-generic-array = "0.14.1"
+array-init = { version = "1.0", default-features = false, features = ["const-generics"] }
```

Comment:
We still have to pass this feature, since this dependency will only make it available be default once `min_const_generics` is in a stable release.

Reply:
hmmm good point.
best would be to sync `array-init` with us or vice versa.